.travis.yml

```diff
@@ -1,9 +1,10 @@
 env:
- - CHRONOSVERSION: 2.3.4
  - CHRONOSVERSION: 2.4.0
 
 language: python
-python: 2.7
+python:
+  - 2.7
+  - 3.5
```

Comment:
According to travis, it is using 3.5
https://travis-ci.org/asher/chronos-python/jobs/114209535#L148

Also, my project is in python 3.5, and I am using this piece of code.